index.html

```diff
@@ -11,6 +11,14 @@
         <link rel="icon" href="./images/avatar.png">
         <title>Gerardo's Portfolio</title>
     </head>
+    <!-- Google tag (gtag.js) -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-9E36WY2R8E"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'G-9E36WY2R8E');
+    </script>
```

Comment:
# Great!!! I suggest locating the script into head tags, like Google Analytics indicate

![googleAnalyctics](https://user-images.githubusercontent.com/43626937/231905248-233524db-0982-4202-abaf-bb43321595ca.png)

Reply:
Changes applied!

Thanks...